sale_update_taxes/wizard/update_line_taxes_wizard.py

```diff
@@ -0,0 +1,40 @@
+from odoo import api, fields, models
+
+
+class UpdateLineTaxesWizard(models.TransientModel):
+    _name = 'update.line.taxes.wizard'
+    _description = 'UpdateLineTaxes'
+
+    tax_ids = fields.Many2many(
+        comodel_name='account.tax', domain="[('type_tax_use', '=', 'sale')]",
+        string='Taxes')
+
+    sale_order_id = fields.Many2one(
+        comodel_name='sale.order',
+        string='Sale order',
+    )
+
+    sale_order_line_ids = fields.One2many(
+        comodel_name='update.line.taxes.wizard.line',
+        inverse_name='update_line_taxes_id',
+        string='Lines',
+        required=False)
+
+    def action_update_taxes(self):
+        for rec in self:
+            for line in rec.sale_order_line_ids:
+                if line.is_update_tax:
+                    line.sale_order_line_id.write({'tax_id': [(6, 0, rec.tax_ids.ids)]})
```

Comment:
There is a shorter way to do this, but it's an "Odoo" way so it's normal if you are not aware of this. Also, you are in an action that is always called on a single model, so no need for a loop.

```suggestion
    def action_update_taxes(self):
        self.ensure_one()
        self.sale_order_line_ids.filtered('is_update_tax').sale_order_line_id.tax_id = self.tax_ids.ids
```

Maybe check that it is correct before modifying, as I didn't test it myself 

Thanks